cypress/integration/integration_spec.js

```diff
@@ -562,51 +554,19 @@ describe('awards for all', function() {
         }
 
         function stepProjectCosts() {
-            function addItem(description, amount) {
+            const amounts = new Array(random(3, 10))
+                .fill(null)
+                .map(() => random(100, 1000));
+
+            amounts.forEach((amount, index) => {
                 cy.getAllByTestId('budget-row')
                     .last()
                     .within(() => {
-                        cy.getByLabelText('Item or activity').type(description);
-                        cy.getByLabelText('Amount').type(amount);
+                        cy.getByLabelText('Item or activity').type(
+                            `Example budget item ${index + 1}`
+                        );
+                        cy.getByLabelText('Amount').type(amount.toString());
                     });
-            }
-
-            function deleteLastRow() {
```

Comment:
I've deleted this error code as we have coverage for this in other tests and it's the only place in these tests where we check that. Can reinstate if it's felt these have more value.